listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -144,11 +144,9 @@ def get_timestamps_for_user(self, user_name):
             t0 = time.monotonic()
             min_ts = self._select_single_timestamp(True, user_name)
             max_ts = self._select_single_timestamp(False, user_name)
```

Comment:
can we add to the docstring of select_single_timestamp to say that it returns a sentinel 0 in the case that they user has no listens?